apps/remix-ide/src/app/panels/main-view.js

```diff
@@ -65,20 +65,20 @@ export class MainView {
       self._view.mainPanel.style.display = 'none'
       self._components.contextView.show()
     })
-    self.tabProxy.event.on('openFile', (file) => {
```

Comment:
we can't call ON here, because the tabproxy plugin is not registered yet. plugin won't allow that.... what to do? same for all other on calls, can only be made after the plugin is loaded into the engine.